pkg/option/config.go

```diff
@@ -2414,6 +2421,13 @@ func (c *DaemonConfig) Populate() {
 
 	c.populateDevices()
 
+	if c.IPAM == ipamOption.IPAMOperator {
+		var nativeCIDR string
+		c.IPAMClusterPoolIPv4RoutingCIDR = viper.GetStringSlice(operatorOption.IPAMOperatorV4CIDR)
+		if nativeCIDR = strings.Join(c.IPAMClusterPoolIPv4RoutingCIDR[:0], ""); nativeCIDR != "" {
```

Comment:
and we can initialize the variable here `if nativeCIDR := strings...`

Reply:
Done.